### ISSUE TITLE
Add label parameters to the confusion matrix function

### ISSUE THREAD

**Issue by saskra** [talboger/fastdist#18]
https://github.com/talboger/fastdist/blob/a7f21b86f81acfbae7bf94c6a4d105ab0744efeb/fastdist/fastdist.py#L1250

I would like to use your function to calculate the confusion matrix because it is twice as fast as scikit-learn's on my segmented images, but unfortunately I am missing an important parameter: Namely, I need the "labels" option because I don't have all classes represented in each example, yet I would like to compute the confusion matrix comparably for each image, each fold of the cross-validation and the overall dataset, see https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html . Could you extend your function accordingly?

Here is a small example script to understand the problem.
```python
import numpy as np
from fastdist import fastdist
from sklearn import metrics

labels = np.asarray([0, 1, 2, 3])
print('Simple vectors with all classes in both ones')
y_true1 = np.asarray([0, 1, 2, 3])
print('True classes: ' + str(y_true1))
y_pred1 = np.asarray([0, 1, 2, 3])
print('Predicted classes: ' + str(y_pred1))
cm1 = fastdist.confusion_matrix(y_true1, y_pred1).astype(np.uint64)
print('Fastdist confusion matrix: \n' + str(cm1))
cm2 = metrics.confusion_matrix(y_true1, y_pred1)
print('Scikit-learn confusion matrix: \n' + str(cm2))
print('Are the resulting matrices identical? ' + str(np.unique((cm1 == cm2))) + '\n')

print('In each vector another class is missing')
y_true1 = np.asarray([0, 1, 2, 0])
print('True classes: ' + str(y_true1))
y_pred1 = np.asarray([3, 1, 2, 3])
print('Predicted classes: ' + str(y_pred1))
cm1 = fastdist.confusion_matrix(y_true1, y_pred1).astype(np.uint64)
print('Fastdist confusion matrix: \n' + str(cm1))
cm2 = metrics.confusion_matrix(y_true1, y_pred1)
print('Scikit-learn confusion matrix: \n' + str(cm2))
print('Are the resulting matrices identical? ' + str((cm1 == cm2)) + '\n')

print('The same class is missing in both vectors, labels parameter used')
y_true1 = np.asarray([0, 1, 0, 3])
print('True classes: ' + str(y_true1))
y_pred1 = np.asarray([0, 1, 0, 3])
print('Predicted classes: ' + str(y_pred1))
cm1 = fastdist.confusion_matrix(y_true1, y_pred1).astype(np.uint64)
print('Fastdist confusion matrix: \n' + str(cm1))
cm2 = metrics.confusion_matrix(y_true1, y_pred1, labels=labels)
print('Scikit-learn confusion matrix: \n' + str(cm2))
print('Are the resulting matrices identical? ' + str((cm1 == cm2)) + '\n')

print('The same class is missing in both vectors, labels parameter NOT used')
y_true1 = np.asarray([0, 1, 0, 3])
print('True classes: ' + str(y_true1))
y_pred1 = np.asarray([0, 1, 0, 3])
print('Predicted classes: ' + str(y_pred1))
cm1 = fastdist.confusion_matrix(y_true1, y_pred1).astype(np.uint64)
print('Fastdist confusion matrix: \n' + str(cm1))
cm2 = metrics.confusion_matrix(y_true1, y_pred1)
print('Scikit-learn confusion matrix: \n' + str(cm2))
print('Are the resulting matrices identical?\n' + str((cm1 == cm2)) + '\n')
```

**Comment by talboger** [comment]
good point, this has been fixed in v1.1.6 (https://github.com/talboger/fastdist/commit/f4c03a706e955fad0955ec316c943fe250b5e2ec)
